AncientReligions/history/characters/irish.txt

```diff
@@ -23509,7 +23509,10 @@
 	culture="irish"
 	father = 7258
 	550.1.1={
-		birth="550.1.1" 
+		birth="550.1.1"
+		effect = {
+			set_global_flag = ancrel
```

Comment:
Sorry, what I meant is I don't think the flag is checked against somewhere else, so I thought maybe there was a typo ? The name doesn't say what this flag is used for :)